gradle.properties

```diff
@@ -9,6 +9,9 @@
 android.enableJetifier=true
 android.useAndroidX=true
 org.gradle.jvmargs=-Xmx1536m
+org.gradle.parallel=true
+org.gradle.caching=true
+kotlin.incremental=true
```

Comment:
Did we check if all our annotation processors are incremental?

Reply:
As far as I know if they are not, the compilation simply won't be incremental, no harm.

Reply:
Although... the kotlin.incremental is already true by default, in not changing it there: https://blog.jetbrains.com/kotlin/2017/03/kotlin-1-1-1-is-out/

Reply:
I'll remove the `kotlin.incremental` as it is already default behaviour, it is not adding anything.